migrations/bin/cli

```diff
@@ -19,5 +19,5 @@ module Migrations
     end
   end
 
-  CommandLineInterface.start
+  Dir.chdir(Rails.root) { CommandLineInterface.start } # rubocop:disable Discourse/NoChdir
```

Comment:
I’m adding this higher up rather here instead of specifically to the uploads command because I think it’s more intuitive for all the commands to behave consistently.